src/screens/DataApi/DataCoverageTable.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableRow from '@material-ui/core/TableRow';
+
+import CheckIcon from '@material-ui/icons/Check';
+
+import {
+  TextWithTooltipContainer,
+  UnderlinedText,
+  StyledTableHead,
+  MetricTableCell,
+  AvailabilityTableCell,
+  MetricTableRow,
+  MetricTableCellHeader,
+} from './DataCoverageTable.style';
+import coverageSummary from './coverage-summary.json';
+import { formatPercent } from 'common/utils';
+import { InfoTooltip } from 'components/InfoTooltip';
+import { pluralize } from 'components/Explore/utils';
+import { RegionType } from 'common/regions';
+
+export const COVERAGE_SUMMARY = coverageSummary as MetricCoverage[];
+
+interface MetricField {
+  displayName: string;
+  field: string;
+}
+
+export const FIELDS: MetricField[] = [
+  { displayName: 'Overall risk level', field: 'riskLevels.overall' },
+  { displayName: 'Daily new cases', field: 'metrics.caseDensity' },
+  { displayName: 'Infection rate', field: 'metrics.infectionRate' },
+  { displayName: 'Positive test rate', field: 'metrics.testPositivityRatio' },
+  { displayName: '% Vaccinated', field: 'metrics.vaccinationsInitiatedRatio' },
+  { displayName: 'ICU capacity used', field: 'metrics.icuCapacityRatio' },
+  { displayName: 'Hospitalization usage', field: 'metrics.icuCapacityRatio' },
+  { displayName: 'Deaths', field: 'actuals.deaths' },
+  { displayName: 'Total testing volume', field: 'actuals.positiveTests' },
+];
+
+export const FIELDS_PRETTY_NAMES = FIELDS.map(field => {
+  const record = COVERAGE_SUMMARY.find(record => field.field === record.name);
+  if (record) {
+    return {
+      ...record,
+      ...{ name: field.displayName },
+    };
+  }
+
+  return null;
+}).filter((record): record is MetricCoverage => record !== null);
+
+export interface CoverageDetails {
+  totalRegions: number;
+  regionsAvailable: number;
+  totalStates: number;
+  statesAvailable: number;
+}
+
+export interface MetricCoverage {
+  name: string;
+  state: CoverageDetails;
+  county: CoverageDetails;
+  metro: CoverageDetails;
+}
+
+const TextWithTooltip: React.FC<{ text: string; tooltipContent: string }> = ({
+  text,
+  tooltipContent,
+}) => {
+  return (
+    <TextWithTooltipContainer>
+      <UnderlinedText align="center">
+        {text}
+        <InfoTooltip trackOpenTooltip={() => {}} title={tooltipContent} />
+      </UnderlinedText>
+    </TextWithTooltipContainer>
+  );
+};
+
+const tooltipText = (coverage: CoverageDetails, regionType: RegionType) => {
+  switch (regionType) {
+    case RegionType.COUNTY:
+      return (
+        `Supported in ${coverage.regionsAvailable} counties (of ${coverage.totalRegions}), ` +
+        `or ${coverage.statesAvailable} states and territories (of ${coverage.totalStates}).`
+      );
```

Comment:
nit: I think you don't need to concatenate the strings when using template strings, you can just have a long line

Reply:
ah yeah I split them somewhat intentionally because I think it's easier to read. Do we have a pattern of not doing so?